e2e/coverage-provider-v8/no-sourcemap/package.json

```diff
@@ -6,6 +6,9 @@
     "transform": {
       "\\.[jt]sx?$": "babel-jest",
       "\\.css$": "<rootDir>/cssTransform.js"
-    }
+    },
+    "coveragePathIgnorePatterns": [
+      "cssTransform\\.js"
+    ]
```

Comment:
This looks breaking for me. See alternative proposal in #13974

Reply:
I'm good with your proposal. Thanks!